_episodes/02-markdown.md

```diff
@@ -221,8 +221,46 @@ keypoints:
 {: .challenge }
 
 
+Linking other websites is an essential feature of a website.
+Bare URLs like https://carpentries.org/ can be made clickable by wrapping them into 
+angle brackets like this <https://carpentries.org/>.
+However, usually the bare URL is not very nice and displaying a clickable
+alternative text is beneficial.  
+You can define either
+1. Inline style links - specifying the URL in line - or
+2. Reference style links - referencing a reusable link reference -
 
 
+> ## Links
+> Fill in the blanks below to create this rendered view
+> 
+> Examples:
+> 1. [Inline link](https://carpentries.org/)
+> 2. [Reference link][case-InSeNsiTiVe-reference-tag]
+>
+> [case-insensitive-reference-tag]: https://carpentries.org/
```

Comment:
I'm confused about the purpose of this section?  It seems to be repeated in the solution?

Reply:
Thank for looking at this!

My intention is to have these three stages:
1. Rendered
2. Fill-in-the-blanks
3. Source

![image](https://user-images.githubusercontent.com/55383008/91875742-335c2c80-ec7c-11ea-91a3-0deb71295b6b.png)

Reply:
That makes sense! Apologies for my confusion